### DISCUSSION TITLE
Shear Book Sprint 1 : PSF

### DISCUSSION THREAD

**Discussion by sfarrens** [CosmoStat/Shear-and-PSF-Reading-Group#54] [Sprint]
## Shear Book :book: Sprint Friday (16/07/21)

We decided that we will focus this sprint on the PSF section of the book. The idea being to have a complete first version of the whole section by the end of the day, which will serve as a template for further development. If this sprint is successful, we can then envision organising future sprints to finalise other sections of the book. In particular, we plan to work on the following tasks:

- [ ] Review and finalise @Thomas Sainrat’s PR on rho statistics (see #52). :chart_with_downwards_trend: 
- [ ]  Clean up and finalise the introduction to the PSF. :broom: 
- [ ]  Write a page on star selection (mainly focused on identifying a stellar locus in a size-magnitude plane). :star: 
- [ ]  Write a page on parametric PSF modelling. :telescope:
- [ ]  Write a page on non-parametric PSF modelling. :stars: 
- [ ]  Write a page (or some pages) on PSF validation methods. :bar_chart:
- [ ]  Resolve some technical issues with the examples and the book itself. :wrench:

**Comment by thuiop** [comment]
My github username is @thuiop for reference, not thomas ^^
